dotenv/src/parse.rs

```diff
@@ -173,7 +173,7 @@ fn parse_value(input: &str, substitution_data: &mut HashMap<String, Option<Strin
 
     //XXX also fail if escaped? or...
     if substitution_mode == SubstitutionMode::EscapedBlock || strong_quote || weak_quote {
-        Err(Error::LineParse(input.to_owned()))
+        Err(Error::LineParse(input.to_owned(), input.len() - 1))
```

Comment:
I know that according to the nature of the code above `input.len()` here could never be 0. But it would be nice to see code here for handling that just in case.

Reply:
Well, why not, done.